jest.config.js

```diff
@@ -126,13 +126,16 @@ module.exports = {
   setupFiles: [],
 
   // A list of paths to modules that run some code to configure or set up the testing framework before each test
-  setupFilesAfterEnv: ["jest-extended"],
+  setupFilesAfterEnv: [
+    "jest-extended",
+    "@testing-library/jest-dom/extend-expect"
+  ],
 
   // A list of paths to snapshot serializer modules Jest should use for snapshot testing
   // snapshotSerializers: [],
 
   // The test environment that will be used for testing
-  testEnvironment: "node",
+  testEnvironment: "jsdom",
```

Comment:
What do you think if we create a separate Jest config file that extends this one and overrides anything that needs overriding?

I'm also thinking that some of the settings here probably shouldn't apply to the UI. For instance, I write code with TDD so it's easy for me to reach 100% unit test coverage, but things are different on the UI and I have no expectation to reach that level of coverage in the UI, so it wouldn't be right to require that.

Reply:
This is done.